File: BlazorApp1/Models/ProgramService.cs
using System.Collections.Generic;

namespace BlazorApp1.Models
{
    public class ProgramService
    {
        private readonly List<SchoolPrograms> programs;

        public ProgramService()
        {
            programs =
            [
                new SchoolPrograms("Computer Science"),
                new SchoolPrograms("Mathematics"),
                new SchoolPrograms("Physics"),
                new SchoolPrograms("Chemistry"),
                new SchoolPrograms("Biology"),
                new SchoolPrograms("Engineering"),
                new SchoolPrograms("Business Administration"),
                new SchoolPrograms("Psychology")
            ];
        }

        public List<SchoolPrograms> GetAllPrograms()
        {
            return programs;
        }
    }
}

File: PythonEnrollmentSystem/program_service.py
from school_programs import SchoolPrograms

class ProgramService:
    def __init__(self):
        self.programs = [
            SchoolPrograms("Computer Science"),
            SchoolPrograms("Mathematics"),
            SchoolPrograms("Physics"),
            SchoolPrograms("Chemistry"),
            SchoolPrograms("Biology"),
            SchoolPrograms("Engineering"),
            SchoolPrograms("Business Administration"),
            SchoolPrograms("Psychology")
        ]

    def get_all_programs(self):
        return self.programs